mava/wrappers/jumanji.py

```diff
@@ -372,3 +379,117 @@ def observation_spec(self) -> specs.Spec[Union[Observation, ObservationGlobalSta
             return specs.Spec(ObservationGlobalState, "ObservationSpec", **obs_data)
 
         return specs.Spec(Observation, "ObservationSpec", **obs_data)
+
+
+class MultiCVRPWrapper(MultiAgentWrapper):
+    """Wrapper for MultiCVRP environment."""
+
+    def __init__(self, env: MultiCVRP, add_global_state: bool = False):
+        env.num_agents = env._num_vehicles
+        env.time_limit = env._num_customers + 1  # added for consistency
+        env.action_dim = env._num_customers + 1  # n_costumers + 1 starter node
+        self.has_global_state = add_global_state
+        self.num_customers = env._num_customers
+        self._env = env
+        super().__init__(env, False)
```

Comment:
I removed this in my last comment because there's no need to initialize the environment since the parent class does it for you.

```suggestion
        super().__init__(env, False)
```